### PR TITLE
adding final script to compare extracted data to reference data and apply unique identifier

### PR DESCRIPTION
**Pull request by esilgard**
- compare reference data (from Reid Lab) to extracted data (from OCR)
- if dates and some combination of 'RS' numbers match : then apply unique sort number identifier to record

### CODE REVIEW

File: match_sorts_to_ref.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            
+            return ret  
+    
+    extracted.apply(get_ret, axis=1)   
+    return extracted
+
+
+def parse_arguments(parser):
+    parser.add_argument('--dir', type=str, help='base directory',
+            default='C:/Users/esilgard/Fred Hutchinson Cancer Research Center/Sanchez, Carissa A - SortScanning')
```
Comment (ddong63) [inline match_sorts_to_ref.py:33]: do you want to point it to data-science folder?

File: match_sorts_to_ref.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+import argparse
+from dateutil.parser import *
```
Comment (ddong63) [inline match_sorts_to_ref.py:4]: ha, i learned something! 🥇

**Review by ddong63 — APPROVED** [review]
lgtm

File: match_sorts_to_ref.py (hunk 1 above)
Comment (esilgard) [reply match_sorts_to_ref.py:33]: no -the data doesn't live in the data science folder, it's in the Reid lab One Drive - mapped to my C atm

File: match_sorts_to_ref.py (hunk 2 above)
Comment (esilgard) [reply match_sorts_to_ref.py:4]: oh - the dateutil parser? it's pretty handy. I don't think I actually implemented it in this script tho....

**Pull request merged**